manifests/kustomize/base/cache-deployer/kustomization.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - cache-deployer-clusterrole.yaml
-  - cache-deployer-clusterrolebinding.yaml
   - cache-deployer-secret-clusterrole.yaml
```

Comment:
My only request left is to change this to a role instead.

Reply:
Thanks, Yuan! The reason using clusterrole here is that if we had multiple cache-deployer instance we only have one clusterrole object. And using rolebinding with clusterrole will also only limit the permission to namespace only. But here is not much difference so change it back to role. Thanks for your quick review! :)

Reply:
Yes, your understanding is right.

My preference came from, it's actually better to let each namespace has its own role than sharing the same cluster role. There could be version mismatch between namespaces. Fewer shared resources mean better flexibility.